src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/FlatFWTest.java

```diff
@@ -61,57 +71,108 @@ public void shouldProvideTypeId() throws Exception
     @Test
     public void shouldDefaultValues() throws Exception
     {
-        int limit = flatRW.wrap(buffer, 0, 100)
+        final int offset = 11;
+        int expectedLimit = setAllRequiredBufferValues(expected, offset);
+
+        Builder builder = flatRW.wrap(buffer, offset, expectedLimit);
```

Comment:
Since there is no terminal octets field we should pass in buffer.capacity() instead of expectedLimit. Please make similar change in the other tests where applicable.